admin-http-gateway/src/main.rs

```diff
@@ -121,12 +121,13 @@ fn metrics(state: &rocket::State<State>) -> Result<String, String> {
 
 #[rocket::main]
 async fn main() -> Result<(), rocket::Error> {
+    let (logger, _global_logger_guard) = create_app_logger(o!());
```

Comment:
Looking at `create_app_logger()` does sentry need to be initialized first? https://github.com/mobilecoinfoundation/mobilecoin/blob/master/common/src/logger/loggers/mod.rs#L227

It's unfortunate there's these temporal dependencies, it looks like it needs to be
1. sentry
2. logger
3. panic handler

But I'm not familiar enough to know if sentry matters

Reply:
sentry doesn't need to be active. If it's not configured then this will be a no-op. Not all apps we have enable sentry. 
For example, https://github.com/mobilecoinfoundation/mobilecoin/blob/master/consensus/mint-client/src/bin/main.rs#L25 does not enable sentry (we don't generally enable it for applications not running on our k8 environment) but logging still works okay. Additionally, `mc_common::sentry::init()` is a [no-op](https://github.com/mobilecoinfoundation/mobilecoin/blob/master/common/src/sentry.rs#L48) when the `MC_SENTRY_DSN` environment variable is not provided.

Some other relevant references:
https://github.com/mobilecoinfoundation/mobilecoin/blob/master/common/src/logger/loggers/sentry_logger.rs#L14
https://github.com/getsentry/sentry-rust/blob/a77bdcefd7c3e976391587549e1a5818a192821b/sentry-core/src/api.rs#L41
https://github.com/getsentry/sentry-rust/blob/master/sentry-core/src/hub.rs#L172